plenum/__init__.py

```diff
@@ -5,12 +5,40 @@
 from __future__ import absolute_import, division, print_function
 
 import sys
-import plenum
-from plenum.common.jsonpickle_util import setUpJsonpickle
-
-from .__metadata__ import *  # noqa
-
 if sys.version_info < (3, 5, 0):
     raise ImportError("Python 3.5.0 or later required.")
 
+import os   # noqa
+from importlib.util import module_from_spec, spec_from_file_location    # noqa: E402
+
+import plenum   # noqa: E402
+import plenum.server.plugin     # noqa: E402
+from plenum.config import ENABLED_PLUGINS   # noqa: E402
+
+
+PLUGIN_LEDGER_IDS = set()
+PLUGIN_CLIENT_REQUEST_FIELDS = {}
+
+
+def setup_plugins():
+    global PLUGIN_LEDGER_IDS
+    for plugin_name in ENABLED_PLUGINS:
```

Comment:
Do we have tests for ENABLED_PLUGINS functionality?

Reply:
The tests are present in the actual plugin packages